ApiSurface/VersionFile.fs

```diff
@@ -2,34 +2,43 @@ namespace ApiSurface
 
 open System.IO
 open System.IO.Abstractions
-open Newtonsoft.Json
+open System.Text.Json
 open System.Reflection
+open System.Text.Json.Serialization
 
 /// A record representing the layout of a version.json file, e.g. as consumed by NerdBank.GitVersioning.
 type VersionFile =
     {
-        /// The version number (e.g. "1.0.1")
-        [<JsonProperty("version")>]
+        /// The version number (e.g. "1.0")
```

Comment:
(It is possible to specify a patch version here, but it's definitely not the mainline case, I imagine.)